src/mlpack/methods/time_series_forecast/exponential_smoothing/ses.cpp

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file methods/time_series/exponential_smoothing/ses.cpp
+ * @author Aditi Pandey
+ *
+ * Implementation of Single exponential smoothing.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#include <mlpack/core/util/log.hpp>
+#include "ses.hpp"
+#include <cmath>
+#include <numeric>
+
+using namespace mlpack;
+using namespace mlpack::ts;
+using namespace mlpack::util;
+using namespace arma;
+using namespace std;
+
+// Calculate errors
+
+double CalcMae(arma::rowvec& errors) {
```

Comment:
I think we already have utilities to compute these quantities in `src/mlpack/core/cv/metrics/` (or somewhere, I can't remember exactly where).  If you search for MAE and MSE in the codebase, you should be able to find a version that you can use instead. :+1:

Reply:
Awesome, I'll look into it.